packages/expo-updates/e2e/setup/project.ts

```diff
@@ -20,7 +20,8 @@ function getExpoDependencyChunks({
     ['@expo/config-types', '@expo/env'],
     ['@expo/config'],
     ['@expo/config-plugins'],
-    ['@expo/cli', 'expo', 'expo-asset', 'expo-modules-core', 'expo-modules-autolinking'],
+    ['expo-modules-core'],
+    ['@expo/cli', 'expo', 'expo-asset', 'expo-modules-autolinking'],
```

Comment:
we need to pack expo-modules-core before expo since expo has some typescript code depends on expo-modules-core.